service/controller/v15/key/key.go

```diff
@@ -36,14 +39,18 @@ const (
 	// SuccessThreshold is SuccessThreshold param in liveness probe config
 	SuccessThreshold = 1
 
+	// Environment variable names for Downward API use (shutdown-deferrer).
+	EnvKeyMyPodName      = "MY_POD_NAME"
+	EnvKeyMyPodNamespace = "MY_POD_NAMESPACE"
+
 	FlannelEnvPathPrefix = "/run/flannel"
 	CoreosImageDir       = "/var/lib/coreos-kvm-images"
 	CoreosVersion        = "1688.5.3"
 
 	K8SEndpointUpdaterDocker = "quay.io/giantswarm/k8s-endpoint-updater:df982fc73b71e60fc70a7444c068b52441ddb30e"
-	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:690c2ffed5d610689345da335175c6cc840ec308"
+	K8SKVMDockerImage        = "quay.io/giantswarm/k8s-kvm:648853b8a0e804eb06738438a2a5f21c673f5be4"
 	K8SKVMHealthDocker       = "quay.io/giantswarm/k8s-kvm-health:ddf211dfed52086ade32ab8c45e44eb0273319ef"
-	ShutdownDeferrerDocker   = "quay.io/giantswarm/shutdown-deferrer:dd07d71449651b2e9c8bd5d7834100451984b722"
+	ShutdownDeferrerDocker   = "quay.io/giantswarm/shutdown-deferrer:fc61a09a64b72e184033a74bfd67b3853241d54a"
```

Comment:
Let's make sure these image tags are the ones we want to get into the version before merging.

Reply:
Done.